src/combine.cpp

```diff
@@ -3832,3 +3788,65 @@ static void handle_java_assert(chunk_t *pc)
       }
    }
 }
+
+
+/**
+ * Detect a cpp braced init list
+ */
+bool detect_cpp_braced_init_list(chunk_t *pc, chunk_t *next)
```

Comment:
Please put the function in extra files (.cpp, .h)
Use an #include ...
Modify the file CMakeLists.txt, to take care

Reply:
What shall I name the new files? Any suggestions? Thanks

Reply:
I moved this function into a new file, 'flag_braced_init_list.cpp'